kclvm/runtime/src/api/kclvm.rs

```diff
@@ -311,6 +311,16 @@ pub struct PanicInfo {
     pub is_warning: bool,
 }
 
+impl PanicInfo {
+    #[cfg(target_os = "windows")]
+    pub fn path_to_windows(&mut self) {
+        self.rust_file = self.rust_file.replace("/", "\\");
```

Comment:
Fix the place where the data generates the newline character instead of replacing it here

Reply:
Fixed. I removed that code and changed it to the test case, and now the path in the test case is automatically adjusted for different platforms.